### DISCUSSION TITLE
what is necessary pytorch or tensoflow for custom dataset training on gpu ? can pytorch-directml be used ?

### DISCUSSION THREAD

**Discussion by Akash7789** [coqui-ai/TTS#1106] [General Q&A]
Hi, I want to know if I tensoflow is necessary for training on a gpu as I am working with an integrated intel gpu. If pytorch is used (In case of training) will using pytorch-directml (pytorch built for intel gpus)be any good ? 
Pytorch-directml ver. 1.8
README asks for pytorch => 1.7

**Comment by erogol** [comment]
You need pytorch but I've no idea about direct ml
